portal.go

```diff
@@ -1824,3 +1867,21 @@ func (portal *Portal) HandleNewDisappearingMessageTime(newTimer uint32) {
 		intent.SendNotice(portal.MXID, fmt.Sprintf("Disappearing messages set to %s", exfmt.Duration(time.Duration(newTimer)*time.Second)))
 	}
 }
+
+func (portal *Portal) HasRelaybot() bool {
+	return portal.bridge.Config.Bridge.Relay.Enabled && len(portal.RelayUserID) > 0
+}
+
+func (portal *Portal) addRelaybotFormat(userID id.UserID, content *event.MessageEventContent) bool {
+	member := portal.MainIntent().Member(portal.MXID, userID)
+	if member == nil {
+		member = &event.MemberEventContent{}
+	}
+	content.EnsureHasHTML()
+	data, err := portal.bridge.Config.Bridge.Relay.FormatMessage(content, userID, *member)
+	if err != nil {
+		portal.log.Error().Msgf("Failed to apply relaybot format: %s", err)
```

Comment:
Using `.Msgf` is not allowed (unfortunately we forgot to tell Scott before he wrote the bridge)

```suggestion
		portal.log.Err(err).Msg("Failed to apply relaybot format")
```

(loggers should also be passed through contexts with fancy context parameters, like event id and stuff in event handlers, but that needs more refactoring)